test/integration/routes_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RoutesTest < ActionController::TestCase
+  test 'APIv1 routes are covered with RBAC enforcement' do
```

Comment:
Let's create a separate test case for each route, with controllers as contexts please. This way we would have a failure for each route if multiple ones are failing.